cool-core/src/test/java/com/nus/cool/core/refactor/CohortStoreTest.java

```diff
@@ -0,0 +1,65 @@
+package com.nus.cool.core.refactor;
+
+import com.google.common.io.Files;
+import com.nus.cool.core.cohort.refactor.storage.CohortRSStr;
+import com.nus.cool.core.cohort.refactor.storage.CohortWSStr;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Testing CohortStoreTest.
+ */
+public class CohortStoreTest {
+
+  /**
+   * Testing cohort query.
+   */
+  @Test(dataProvider = "storeCohortDP")
+  public void storeCohort(ArrayList<String> testData) throws IOException {
+
+    CohortWSStr cws = new CohortWSStr(StandardCharsets.UTF_8);
+
+    for (String userID : testData) {
+      cws.addCubletResults(userID);
+    }
+
+    // output file
+    String fileName = "TestCohort";
+    File cubemeta = new File(System.getProperty("user.dir"), fileName);
```

Comment:
Name the variables other than cubemeta...?

Reply:
Updated to `cohortResFile` also